src/mediaTrackRole.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * The `MediaTrackRole` interface represents the role of a media track in a media stream.
+ */
+export interface MediaTrackRole {
+  /**
+   * The unique identifier for this role instance.
+   * - On Android: Corresponds to the native [`MediaTrackRole.id`](https://cdn.bitmovin.com/player/android/3/docs/player-core/com.bitmovin.player.api.media/-media-track-role/id.html).
+   *  May be undefined.
+   * - On iOS and tvOS: Typically `undefined`, as HLS characteristics do not have inherent IDs in this context.
+   */
+  id?: string;
```

Comment:
On iOS and tvOS the value is always `undefined`, no? Since in `ios/RCTConvert+BitmovinPlayer.swift`, we are never passing this value

```suggestion
  /**
   * The unique identifier for this role instance.
   * - On Android: Corresponds to the native [`MediaTrackRole.id`](https://cdn.bitmovin.com/player/android/3/docs/player-core/com.bitmovin.player.api.media/-media-track-role/id.html).
   *  May be undefined.
   * - On iOS and tvOS: `undefined`, as HLS characteristics do not have inherent IDs in this context.
   */
  id?: string;
```

Reply:
True, you are right as it makes sense to simplify the language here.